docs/ja/features/reactivcore-rules.md

```
@@ -1,11 +1,2321 @@
+# ReactiveCore Rules
 
-# ReactiveCore ルール
+GeonicDB の **ReactiveCore Rules** は、エンティティの変更を自動的に検出し、定義されたルールに基づいてアクションを実行するリアクティブ自動化機能です。Change Streams を介して MongoDB の変更をリアルタイムで監視し、ルール条件が一致したときに自動処理を実行します。
 
-::: info 準備中
-このページは GeonicDB 上流ドキュメントから同期されます。次回ドキュメント同期後に全コンテンツが利用可能になります。
-:::
+## 目次
+
+
+* [概要](#overview)
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)